### PR TITLE
Renamed doiRequest.date to doiRequest.createdDate

### PR DESCRIPTION
**Pull request by janbj**

### CODE REVIEW

**Review by axthosarouris — DISMISSED** [review]

**Review by norrs — APPROVED** [review]

File: src/main/java/no/unit/nva/model/DoiRequest.java [hunk 1]
```diff
@@ -12,7 +12,7 @@
 public class DoiRequest {
 
     private DoiRequestStatus status;
-    private Instant date;
+    private Instant createdDate;
```
Comment (brinxmat) [inline src/main/java/no/unit/nva/model/DoiRequest.java:15]: If we add `@JsonAlias("date")`, we avoid breaking the API

**Review by brinxmat — APPROVED** [review]

**Review by norrs — CHANGES_REQUESTED** [review]
Agree with  @brinxmat , fix the JsonAlias :+1:

**Review by brinxmat — COMMENTED** [review]
Accidentally clicked Approve, see change request

**Review by brinxmat — CHANGES_REQUESTED** [review]
Did it again

**Review by norrs — APPROVED** [review]

**Review by axthosarouris — APPROVED** [review]

**Pull request merged**